test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void isAnyNull_returnsFalse() {
+		final Object[] testInputs = { "first", "second", "third", "fourth" };
+		assertFalse(Utils.isAnyNull(testInputs));
+	}
+	
+	@Test
+	public void isAnyNull_returnsTrue() {
```

Comment:
The middle part of the name (i.e. description of the scenario) missing. e.g. `isAnyNull_oneNullValue_returnsTrue`